web-admin/src/components/projects/invalidations.ts

```diff
@@ -0,0 +1,23 @@
+import { invalidationForMetricsViewData } from "@rilldata/web-local/lib/svelte-query/invalidation";
+import type { QueryClient } from "@tanstack/svelte-query";
+
+export async function invalidateProjectQueries(
```

Comment:
Sorry I missed this the last time, but I'd expect `invalidateProjectQueries` to take a `projectName` as a parameter, not a list of `dashboardNames`.

A few options here, but the most clear might be `invalidateDashboardQueries(queryClient, dashboardName)` and feeding this function to every dashboard in a list via a map function. I think it'd be clear code, then we could also put the invalidation function in the same place as the others: `lib/svelte-query/invalidation.ts`

Reply:
I would think there would be more added here like the project status request as well.

Reply:
My main point is the function name & its input parameters are confusing. I'd expect probably one of `invalidateProjectQueries(projectName)`, `invalidateDashboardQueries(dashboardName)`, or `invalidateDashboardsQueries(dashboardNames)`